INSTALL.md

```diff
@@ -41,7 +41,7 @@ d. Install mmdetection (other dependencies will be installed automatically).
 
 ```shell
 python setup.py develop
-# or "pip install -e ."
+# or "pip install -e -v ."
```

Comment:
`-v` cannot be between `-e` and `.`, otherwise there will be an error.

Reply:
Sorry, sometimes I think I know what I am doing and that I can just skip testing 😄